python/tskit/drawing.py

```diff
@@ -221,7 +235,355 @@ def remap_style(original_map, new_key, none_value):
         return str(text_tree)
 
 
-class SvgTreeSequence:
+def add_class(attrs_dict, classes_str):
+    """Adds the classes_str to the 'class' key in attrs_dict, or creates it"""
+    try:
+        attrs_dict["class"] += " " + classes_str
```

Comment:
Why not use a defaultdict here for these attrs_dicts?

Reply:
Doh, of course. I think this was how it was set up previously, so it's a historical thing. I'll change it.

Reply:
Ah, I've just had a look, and these dicts contain a mix of strings and floats/ints, as they are passed as kwargs to svgwrite. I've just tried changing them to defaultdict(str) and it makes svgwrite bomb out. So I think we should leave it as-is.